src/fides/api/models/location_regulation_selections.py

```diff
@@ -146,6 +176,38 @@ async def get_selected_locations_async(
                 return set(record.selected_locations)
             return set()
 
+    @classmethod
+    def get_selected_location_groups(
+        cls,
+        db: Session,
+    ) -> Set[str]:
+        """
+        Utility method to get the selected_locations_groups, returned as a Set.
+
+        Location Groups aren't saved directly, but as a side-effect of saving locations
+        """
+        record = db.query(cls).first()
+        if record:
+            return set(record.selected_location_groups)
+        return set()
+
+    @classmethod
+    async def get_selected_location_groups_async(
+        cls,
+        async_session: AsyncSession,
+    ) -> Set[str]:
+        """
+        Utility method to get the selected_location_groups, returned as a Set.
+
+        Location Groups aren't saved directly, but as a side-effect of saving locations
```

Comment:
i appreciate the diligence in the docstrings here! helps provide context throughout 👍